apis/logging/v1/output_types.go

```diff
@@ -286,7 +286,7 @@ type Http struct {
 
 	// Timeout specifies the Http request timeout in seconds. If not set, 10secs is used.
 	// +optional
-	Timeout string `json:"timeout,omitempty"`
+	Timeout int `json:"timeout,omitempty"`
```

Comment:
Should be `TimeoutSeconds` for clarity. IIRC this is in the k8s API guidelines for time variables. 
Not a blocker if it's too late to change.

Reply:
The http type was released for 5.7 so there is not a place to change it.  I have concerns about changing the time as-is

Reply:
Can we release the hold?